iot/api-client/beta-features/commands/receive/receive_test.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import io
+import os
+import sys
+import time
+
+
+# Add manager for bootstrapping device registry / device for testing
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'send'))  # noqa
+from google.cloud import pubsub
+import pytest
+import send
+
```

Comment:
Nit, why the blank line between send & receive? Also receive should come before send, alphabetically.

Reply:
Potentially a bug in Nox, receive is interpreted as 3P, send is interpreted as local